boards/raspberrypi/rpi_pico2/rpi_pico2.yaml

```diff
@@ -0,0 +1,19 @@
+identifier: rpi_pico2
```

Comment:
this should actually be the full identifier, with soc and CPU cluster, since we already know this has multiple cores which presumably will be supported and will make things much easier rather than breaking everything later on

Reply:
Had a quick look and can see this has not been addressed, my review has not been re-requested so comments are still pending and need addressing

Reply:
What is the exact change that is being requested here? 

> breaking everything later on

Can you be more specific? It might be that users can read the migration guide if/when required, and authors can ensure that they keep the migration guide in a good state.